benchexec/resources.py

```diff
@@ -15,191 +15,199 @@
 import math
 import os
 import sys
+from typing import Optional, List, Dict
 
 from benchexec import cgroups
 from benchexec import util
 
+sys.dont_write_bytecode = True  # prevent creation of .pyc files
+
 __all__ = [
     "check_memory_size",
     "get_cpu_cores_per_run",
     "get_memory_banks_per_run",
     "get_cpu_package_for_core",
 ]
 
+# typing defintions
+ListOfIntLists = List[List[int]]
```

Comment:
`ListOfIntLists` ist used only once:
```
def get_memory_banks_per_run(coreAssignment, cgroups) -> Optional[ListOfIntLists]:
    """Get an assignment of memory banks to runs that fits to the given coreAssignment,
    i.e., no run is allowed to use memory that is not local (on the same NUMA node)
    to one of its CPU cores."""
```
It may be longer, but it doesn't act up in CICD like the predecessor:
```
./benchexec/resources.py:943:6: TAE002 too complex annotation (4 > 3)
) -> Optional[List[List[int]]]:
```
Please suggest a useful name for it?

Reply:
There is no need in complicating the code just to please CI. It would be valuable to assign it a type alias that has a meaningful name, but not `ListOfIntLists`. So if you don't know a better name, then keep it as is (it is pre-existing code, after all), and silence the warning.

Reply:
Renamed `ListOfIntLists` into `_2DIntList`